otelcolbuilder/cmd/01_main.go.patch

```diff
@@ -4,7 +4,7 @@
  	}
 
 -	cmd := otelcol.NewCommand(params)
-+	cmd := otelcol.NewCommandMustSetProvider(params)
```

Comment:
was this is an upstream change?

Reply:
yep that function was deprecated in v107 (at the bottom of breaking changes): https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.107.0

Reply:
also why i asked what is this file on slack 😆 https://sumologic.slack.com/archives/C047C1UM4DC/p1727476645345019